cinema_service/settings.py

```diff
@@ -134,7 +141,7 @@
 STATIC_URL = "static/"
 
 MEDIA_URL = "/media/"
-MEDIA_ROOT = BASE_DIR / "media"
+MEDIA_ROOT = "/files/media"
```

Comment:
The MEDIA_ROOT should be set to a full path using os.path.join(BASE_DIR, 'media'). This way, Django will look for media files in a directory called 'media' inside your project directory.